src/js/components/AppListComponent.jsx

```diff
@@ -314,22 +366,21 @@ var AppListComponent = React.createClass({
       appsHealthCount: getInitialFilterCounts(HealthStatus)
     };
 
-    // Global search view - only display filtered apps
+    // Global search view - only display filtered apps & groups
     if (this.hasFilters()) {
       appListViewType = AppListViewTypes.APP_LIST;
 
-      nodesSequence = this.filterNodes(lazy(state.apps), filterCounts)
+      nodesSequence = lazy(this.filterNodes(state.apps, filterCounts))
```

Comment:
Okay, I see. :)
So, the plan is eventually to _replace_ lazy.js with immutable.js that has also a lazy evaluation.
Would be great, if we could stay with the lazy.js-`nodeSequence` for now, like it was.

Thank you!